lab_2_tokenize_by_bpe/main.py

```diff
@@ -36,6 +56,16 @@ def count_tokens_pairs(
     :param word_frequencies: dictionary in the form of <preprocessed word: number of occurrences>
     :return: dictionary in the form of <token pair: number of occurrences>
     """
+    if not isinstance(word_frequencies, dict):
+        return None
+
+    pair_freq = {}
+    for word in word_frequencies.keys():
+        pair_freq = {(word[i], word[i + 1]): word_frequencies[word] for i in range(len(word) - 1)}
+    #    for i in range(len(word) - 1):
```

Comment:
?

Reply:
At this point I hashed the working code to see if i could make it through a dictionary generator